crates/astria-composer/src/searcher/collector.rs

```diff
@@ -32,6 +32,12 @@ pub(super) struct Transaction {
     pub(super) inner: ethers::types::Transaction,
 }
 
+/// Collector is a sub-actor in the Searcher module that interfaces with
```

Comment:
Please leave an empty doc comment that summarizes this type, e.g.:
```suggestion
/// Collects transactions submitted to a rollup node and passes them downstream for further processing.
///
/// Collector is a sub-actor in the Searcher module that interfaces with
```

Reply:
Yeah, that makes sense